documentation/release_4.0.htm

```diff
@@ -19,7 +19,7 @@ <h2>Overall summary</h2>
         <li>Kris Thielemans (UCL) for general overview, fixes, help and maintenance</li>
         <li>Nikos Efthimiou (UCL and Hull) for the scatter update, list mode reconstruction, ROOT support etc</li>
         <li>Richard Brown (UCL) for improving code structure and safety (e.g. projectors, normalisation, <code>set_up</code> checks),
-          IO additions, handling of dynamic data</li>
+          IO additions, handling of dynamic data, and wrapping of NiftyPET's GPU functionality for projecting and unlisting</li>
```

Comment:
there's a new `release_4.1.htm`...

Reply:
I'm confused, is this a 4.0 or 4.1 feature?

Reply:
4.1, as 4.0 is out already!

Reply:
sure!